base/BaseLoader.vue

```diff
@@ -0,0 +1,34 @@
+<template>
```

Comment:
should be in the root "components" directory

```
/components
  /base
```

Reply:
also using auto import you could keep it even as Loader.vue

https://v3.nuxtjs.org/guide/directory-structure/components#component-names